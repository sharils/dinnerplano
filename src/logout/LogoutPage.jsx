import React from "react";
import { Link } from "react-router-dom";

export default function LogoutPage() {
  return (
    <React.Fragment>
      You have successfully logged out. <Link to="/login">Login again</Link>
    </React.Fragment>
  );
}
