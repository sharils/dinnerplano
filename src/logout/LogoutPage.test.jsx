import React from "react";
import renderer from "react-test-renderer";
import Provizanto from "../../test/util/Provizanto";
import LogoutPage from "./LogoutPage";

it("renders correctly", () => {
  const tree = renderer
    .create(
      <Provizanto>
        <LogoutPage onSubmit={() => {}} />
      </Provizanto>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
