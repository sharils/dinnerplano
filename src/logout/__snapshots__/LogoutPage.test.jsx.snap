// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  "You have successfully logged out. ",
  <a
    href="/login"
    onClick={[Function]}
  >
    Login again
  </a>,
]
`;
